docs/src/main/sphinx/release/release-370.md

```diff
@@ -0,0 +1,62 @@
+# Release 370 (4 Feb 2022)
+
+## General
+
+* Automatically raise the operating system maximum limit for open files to the
```

Comment:
This should be rephrased as "Fix failure when operating system open file count is set too low". Also, this only affects the RPM.

Reply:
Done